### PR TITLE
Fix name typo #trivial

### PR DESCRIPTION
**Pull request by wsdwsd0829**

### CODE REVIEW

**Comment by wsdwsd0829** [comment]
Do we need change log for this kind of simple modifications?

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Any source code changes should have an entry in CHANGELOG.md or have #trivial in their title.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by nguyenhuy** [comment]
@wsdwsd0829 No you don't.

File: Source/ASDisplayNode+Layout.mm [hunk 1]
```diff
@@ -493,10 +493,10 @@ @implementation ASDisplayNode (ASLayoutTransition)
 - (BOOL)_isLayoutTransitionInvalid
 {
   ASDN::MutexLocker l(__instanceLock__);
-  return [self _locked_isLayoutTransitionValid];
+  return [self _locked_isLayoutTransitionInValid];
 }
 
-- (BOOL)_locked_isLayoutTransitionValid
+- (BOOL)_locked_isLayoutTransitionInValid
```
Comment (nguyenhuy) [inline Source/ASDisplayNode+Layout.mm]: Please use lowercase v (e.g "In**v**alid").

**Comment by nguyenhuy** [comment]
Good catch. Thank you, @wsdwsd0829.

**Pull request merged**